CI: `404  Failed to get https://github.com/docker-library/busybox/raw/dist-amd64/stable/glibc/busybox.tar.xz`

```
panic: getImages error exit status 1 (output: curl: (22) The requested URL returned error: 404 
Failed to get https://github.com/docker-library/busybox/raw/dist-amd64/stable/glibc/busybox.tar.xz
)

goroutine 1 [running]:
github.com/opencontainers/runc/libcontainer/integration.init.1()
	/home/runner/work/runc/runc/libcontainer/integration/utils_test.go:37 +0x2c5
FAIL	github.com/opencontainers/runc/libcontainer/integration	0.133s
```

https://github.com/opencontainers/runc/actions/runs/3896184715/jobs/6652404118

Thanks to @egernst for finding this in https://github.com/opencontainers/runc/pull/3698

--- a/tests/integration/get-images.sh
+++ b/tests/integration/get-images.sh
@@ -56,7 +56,7 @@ esac
 # busybox
 BUSYBOX_IMAGE="$TESTDATA/busybox-${arch}.tar.xz"
 get "$BUSYBOX_IMAGE" \
-	"https://github.com/docker-library/busybox/raw/dist-${arch}/stable/glibc/busybox.tar.xz"
+	"https://github.com/docker-library/busybox/raw/dist-${arch}/latest/glibc/busybox.tar.xz"
 echo "BUSYBOX_IMAGE=$BUSYBOX_IMAGE"
 
 # debian

